test/interop/fortran/fortArrayToDRAutomatic/chapelProcs.good

```diff
@@ -0,0 +1,5 @@
+chapelProcs.chpl:8: warning: Unknown Fortran KIND generating interface for C type: _ref_CFI_cdesc_t
+chapelProcs.chpl:8: warning: Unknown Fortran type generating interface for C type: _ref_CFI_cdesc_t
+   10.0000000000000        20.0000000000000        30.0000000000000     
+   40.0000000000000        50.0000000000000        60.0000000000000     
+   70.0000000000000        80.0000000000000        90.0000000000000     
```

Comment:
It seems like the Fortran and Chapel code are thinking of this as a 1D array, but that it's printing as a 2D array?  Or...what am I missing?

Reply:
The Fortran print statement just wraps the output by default. If I change the array to only have 4 elements it prints:
```
   10.0000000000000        20.0000000000000        30.0000000000000
   40.0000000000000
```

Reply:
Great, that's what I'd hoped (though it still surprised me), thanks!